src/mudblazor/MudBlazor.Template/Components/Layout/MainLayout.razor

```diff
@@ -28,9 +29,79 @@
 
 @code {
     private bool _drawerOpen = true;
+    private bool _isDarkMode = true;
 
     private void DrawerToggle()
     {
         _drawerOpen = !_drawerOpen;
     }
-}
\ No newline at end of file
+
+    private void DarkModeToggle()
+    {
+        _isDarkMode = !_isDarkMode;
+    }
+
+    public MudTheme Theme => new MudTheme()
```

Comment:
Should it really use the `=>`? It will be re-evaluated on each call, isn't `=` sufficient here?

Reply:
Good catch, that's fixed